### PR TITLE
Test update concurrent-ruby to 1.1.3

### PR DESCRIPTION
**Pull request by adamruzicka**
Examples seem to work, certain tests get stuck.

### CODE REVIEW

**Comment by iNecas** [comment]
Nice @adamruzicka, let's continue with the tests, but looks promising…

**Comment by adamruzicka** [comment]
This leaves us with two failing tests:
- [ ] `abnormal_states_recovery_test` which currently deadlocks
- [ ] `executor_test` which just hangs

**Comment by adamruzicka** [comment]
Tests are passing for me locally, let's see what Mr. T say. :crossed_fingers:

**Comment by adamruzicka** [comment]
And we're :green_heart:

**Comment by iNecas** [comment]
Very nice. I don't want to get this in right now due to upcoming 1.20 release, but once that's over, we can get this in.

FYI @pitr-ch - the new c-r relesae seems good so far… :)

**Comment by iNecas** [comment]
And I think @mmoll will be also happy about this PR :)

**Comment by adamruzicka** [comment]
That makes sense, I'd like to give it a spin in a production environment before this goes in.

**Comment by iNecas** [comment]
That's a good point: it's also wroth checking if we're not using any of the old API in foreman plugins and if so, ideally having some limited compatibility layer here to allow a smooth transition, if possible

File: dynflow.gemspec [hunk 1]
```diff
@@ -21,8 +21,8 @@ Gem::Specification.new do |s|
   s.add_dependency "multi_json"
   s.add_dependency "apipie-params"
   s.add_dependency "algebrick", '~> 0.7.0'
-  s.add_dependency "concurrent-ruby", '~> 1.0'
-  s.add_dependency "concurrent-ruby-edge", '~> 0.2.0'
+  s.add_dependency "concurrent-ruby", '= 1.1.0.pre2'
+  s.add_dependency "concurrent-ruby-edge", '= 0.4.0.pre2'
```
Comment (mmoll) [inline dynflow.gemspec]: is this still needed?

Comment (iNecas) [reply dynflow.gemspec]: I guess it's needed until the c-r 1.1 is in pre-release

Comment (adamruzicka) [reply dynflow.gemspec]: We still use Actors which live in -edge

Comment (mmoll) [reply dynflow.gemspec]: I see :+1:

**Comment by pitr-ch** [comment]
Great! :) Thanks for trying it out.

**Comment by mmoll** [comment]
What's the status here? :)

**Comment by iNecas** [comment]
After conflicts are resolved and tests are green, I'm ok to hit the merge button

**Comment by adamruzicka** [comment]
And it's :green_heart:

**Comment by iNecas** [comment]
Everything I'v tried seems to work just fine: I don't see any other reason to hold this back any longer. Thanks @adamruzicka

**Comment by iNecas** [comment]
And of course @pitr-ch for great work on concurrent-ruby gem

**Pull request merged**